doc/developer-guide/layout/runroot.en.rst

```diff
@@ -0,0 +1,105 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please follow the heading conventions as used in other files, and documented [here](https://docs.trafficserver.apache.org/en/8.0.x/developer-guide/documentation/conventions.en.html#sections-and-headings).